src/query/service/src/interpreters/common/stream.rs

```diff
@@ -129,7 +136,7 @@ pub async fn query_build_update_stream_req(
         new_table_meta.options = options;
         new_table_meta.updated_on = Utc::now();
 
-        update_table_metas.push(UpdateTableMetaReq {
+        update_table_meta_reqs.push(UpdateTableMetaReq {
```

Comment:
Sometimes we use `UpdateTableMetaReq` to update a stream, and other times we use `UpdateTableMetaReq.update_stream_meta`. But both `SchemaApi::update_table_meta()` and `update_table_meta()` handle `stream` and `table` with subtle differences. Although the current implementation is logically correct, we could consider refactoring in the future to express the requests for updating a stream and updating a table with a more clear structure, to avoid potential bugs.

Reply:
agree